common/src/KokkosKernels_PrintConfiguration.hpp

```diff
@@ -44,6 +44,18 @@ inline void print_cusparse_version_if_enabled(std::ostream& os) {
      << "KOKKOSKERNELS_ENABLE_TPL_CUSPARSE: no\n";
 #endif
 }
+
+inline void print_cusolver_version_if_enabled(std::ostream& os) {
```

Comment:
This function should probably be called below in ``print_enabled_tpls``, under cublas and cusparse